src/install/haproxy/windows/install-msi.mdx

```diff
@@ -0,0 +1,10 @@
+---
+componentType: default
+headingText: Download using MSI
+---
+
+1. Download the latest .MSI installer image for the desired integration [from our repository](https://download.newrelic.com/infrastructure_agent/windows/integrations/).
```

Comment:
ok once i used `desired` in a doc and that gave @austin-schaefer flashbacks to starting out as a tech writer. afaik we try to avoid 'desired' as a turn of phrase in our docs